### DISCUSSION TITLE
strange behavior] Screen turns white when deleting text in the search bar with backspace

### DISCUSSION THREAD

**Discussion by AuroraDBmach** [FreeTubeApp/FreeTube#6430] [Q&A]
1. my Issue
1-1. (User) Launches the FreeTube program.
1-2. (User) Clicks on the search bar and enters 'a' or 'ㄱ' (Korean character).
1-3. (User) Presses the backspace key once quickly to delete the character.
1-4. The screen does not refresh properly and turns white.
1-5. (User) Forced to terminate and restart the program.

2. my Environment Information
2-1. Operating System (OS): Windows 11, version 23H2 (OS Build 22631.4602)
2-2. FreeTube Version: v0.22.0 Beta
2-3. Input Method: Korean IME (Microsoft IME) with both Korean and English languages.

3. Additional Information
- Frequency of Occurrence: The issue occurs every time the backspace key is pressed when the cursor is initially positioned at the beginning of the search input text box.
<img width="562" alt="스크린샷 2024-12-22 194827" src="https://github.com/user-attachments/assets/231b75c5-b258-4a7f-9ffd-9dc757f55dc2" />
<img width="509" alt="스크린샷 2024-12-22 194856" src="https://github.com/user-attachments/assets/8454d215-ae24-43bb-8386-6704c932a302" />
<img width="512" alt="스크린샷 2024-12-22 194927" src="https://github.com/user-attachments/assets/fa48f2b4-626d-4470-99f1-dbf2985d693b" />
<img width="523" alt="스크린샷 2024-12-22 194952" src="https://github.com/user-attachments/assets/5e25ec2a-473d-4cbe-aa2b-160826cd87d2" />
<img width="761" alt="스크린샷 2024-12-22 195019" src="https://github.com/user-attachments/assets/10a4f621-fdca-443e-a0cb-671a6a81cc92" />

**Comment by absidue** [comment]
Duplicate of #6065
